src/components/IssuesList/IssuesList.tsx

```diff
@@ -20,20 +20,21 @@ type IssuesListProp = {
   isLoading: boolean;
   issuesData: DataItem[];
   isUrlProvided: boolean;
+  shouldShowText: boolean;
 };
 const IssuesList: FC<IssuesListProp> = ({
   onIssuePress,
   isLoading,
   issuesData,
   isUrlProvided,
-  route,
+  shouldShowText,
 }) => {
   console.log("IssuesList");
-
+  const { sortingType } = useIssuesContext();
```

Comment:
do we really need `sortingType` to be in a context? It might be better if it was local